netbox_agent/network.py

```diff
@@ -60,7 +60,28 @@ def scan(self):
                 continue
 
             ip_addr = netifaces.ifaddresses(interface).get(netifaces.AF_INET)
-            ip_addr += netifaces.ifaddresses(interface).get(netifaces.AF_INET6)
+            ip6_addr = netifaces.ifaddresses(interface).get(netifaces.AF_INET6)
```

Comment:
There's still the following error:

``` 
File "/tmp/netbox_agent/netbox_agent/network.py", line 80, in scan
    for addr in ip6_addr:
TypeError: 'NoneType' object is not iterable
``` 

Just add this and it'll be good to go!
```suggestion
            ip6_addr = netifaces.ifaddresses(interface).get(netifaces.AF_INET6, [])
```